src/regex/invoices/views.py

```diff
@@ -26,7 +26,7 @@ def get_context_data(self, **kwargs):
             {
                 "tax_rates": tax_rates,
                 "items": self.object.invoiceitem_set.select_related("project").order_by(
-                    "project", "tax_rate"
+                    "date", "project", "tax_rate"
```

Comment:
hrm, not sure what the template looks like again, but maybe swap the order of `project` and `date`, so that items are still grouped by project?

Reply:
Good point, never had a case where I'd have more than a project for an invoice, but makes sense to sort it by project first.